types/src/roller.rs

```diff
@@ -0,0 +1,50 @@
+use crate::base64;
+use blake2::{Blake2s256, Digest};
+use serde_derive::{Deserialize, Serialize};
+use serde_repr::{Deserialize_repr, Serialize_repr};
+
+#[derive(Serialize_repr, Deserialize_repr, PartialEq, Debug)]
```

Comment:
I think this file should be only related to roller. It's better to move to roller

Reply:
Sure